### PR TITLE
XEP-0333 v0.4 - add MUC usage info

### PR DESCRIPTION
**Pull request by mwild1**

### CODE REVIEW

File: xep-0333.xml [hunk 1]
```diff
@@ -236,6 +242,48 @@
...
+</example>
+
+<example caption='Example marker response to a markable MUC message'>
+<![CDATA[
+<message
+    from='coven@chat.shakespeare.lit/secondwitch'
+    to='coven@chat.shakespeare.lit'
+    id='message-2'
+    type='groupchat'
+    >
```
Comment (lnjX) [inline xep-0333.xml:278]:
```suggestion
    type='groupchat'>
```

This fits better to the formatting of the other examples in other XEPs.

Comment (mwild1) [reply xep-0333.xml]: Fixed, thanks!

**Comment by Ppjet6** [comment]
Seems that only half of this requires council. I assume one won't go without the other though.

**Comment by Ppjet6** [comment]
After talking with @mwild1, it seems the commit of #926 is also included in this but both PRs are separate. Only the XEP-0333 should be merged here.

**Comment by Ppjet6** [comment]
This was cherry-picked in 928513d81d3b351f5c10861daeaa59274bb439b2.

**Pull request closed without merging**